### PR TITLE
[11.0] Helpdesk Rating Module

### PR DESCRIPTION
**Pull request by samuel-domatix**
This module gives the possibility of rate by the customer, the assistance received on their tickets.

- Added new menu **Helpdesk / Reports / Customer Rating**
- In the ticket kanban view:
    - Added the percentage of positive ratings of each ticket.
    - Added a new menu on each ticket with its own ratings.
- In the ticket form view:
    - Added new field in the ticket form view to select the type of rating.
    - Added a smartbutton in the ticket form view with all the ratings of the ticket.
- Added new field **Rating Email Template** in stages.
- Modified the tree view of the stages to show the rating email template.
- Created a rating email template.

### CODE REVIEW

File: helpdesk_mgmt_rating/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    'name': 'Helpdesk Management Rating',
+    'summary': """
+        This module allows the customer to rate the assistance received
+        on a ticket.
+        """,
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Domatix',
```
Comment (emagdalenaC2i) [inline helpdesk_mgmt_rating/__manifest__.py]:
```suggestion
    'author': 'Domatix, Odoo Community Association (OCA)',
```

**Review by emagdalenaC2i — CHANGES_REQUESTED** [review]
I think OCA must be as Authors too
See https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py

File: helpdesk_mgmt_rating/__manifest__.py (hunk 1 above)
Comment (samuel-domatix) [reply helpdesk_mgmt_rating/__manifest__.py]: Changed!

**Review by catalin-domatix — APPROVED** [review]

**Review by alvaro-domatix — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
:+1:

**Review by emagdalenaC2i — APPROVED** [review]

File: helpdesk_mgmt_rating/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    'summary': """
+        This module allows the customer to rate the assistance received
+        on a ticket.
+        """,
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Domatix, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/helpdesk',
+    'category': 'Productivity',
```
Comment (pedrobaeza) [inline helpdesk_mgmt_rating/__manifest__.py:12]: Please put the same category than the main module

**Pull request merged**